board/faults.h

```diff
@@ -24,7 +24,7 @@
 #define FAULT_REGISTER_DIVERGENT            (1U << 18)
 #define FAULT_INTERRUPT_RATE_KLINE_INIT     (1U << 19)
 #define FAULT_INTERRUPT_RATE_CLOCK_SOURCE   (1U << 20)
-#define FAULT_INTERRUPT_RATE_TIM9           (1U << 21)
+#define FAULT_INTERRUPT_RATE_HEARTBEAT      (1U << 21)
```

Comment:
Don't agree with the name "heartbeat", we usually use this to refer to the message openpilot sends to panda periodically such that the panda knows it's still "alive".
I suggest something more like "tick" or something, not 100% sure what it should be called

Reply:
I understand, has this in my notes not to forget ;)